### ISSUE TITLE
hyperparameters for finetune ResNet34 for HMDB51

### ISSUE THREAD

**Issue by LiliMeng** [kenshohara/3D-ResNets-PyTorch#70]
Hi,
       Thanks a lot for providing the open-source code!

       May I ask what kind of hyperparameters did you use for fine-tune HMDB51? How many epochs did you train. I used the default parameters for fine-tune UCF101, but after 10 epochs, the training accuracy reached 70%, but the best test accuracy is around 48%. The paper reported 59.1%.

         Highly appreciate your time and help!

**Comment by kenshohara** [comment]
This is my opts.json when finetuning resnet-34 on HMDB-51.

{
    "root_path": "../../../HMDB51/",
    "video_path": "../../../HMDB51/videos/jpg",
    "annotation_path": "../../../HMDB51/hmdb51_1.json",
    "result_path": "../../../HMDB51/results/ft/split1/resnet34_1e-03_1e-05_4",
    "dataset": "hmdb51",
    "n_classes": 400,
    "n_finetune_classes": 51,
    "sample_size": 112,
    "sample_duration": 16,
    "sample_stride": 16,
    "initial_scale": 1,
    "n_scales": 5,
    "scale_step": 0.84089641525,
    "learning_rate": 0.001,
    "momentum": 0.9,
    "weight_decay": 0.00001,
    "nesterov": false,
    "optimizer": "sgd",
    "lr_patience": 10,
    "batch_size": 128,
    "n_epochs": 300,
    "begin_epoch": 1,
    "n_val_samples": 3,
    "resume_path": "",
    "pretrain_path": "../../../HMDB51/models/resnet-34-kinetics.pth",
    "ft_begin_index": 4,
    "no_train": false,
    "no_val": false,
    "test": true,
    "test_subset": "val",
    "test_merge": "avg",
    "test_scale": 1,
    "no_cuda": false,
    "n_threads": 12,
    "checkpoint": 10,
    "crop": "c",
    "no_hflip": false,
    "norm_value": 1,
    "model": "resnet",
    "model_depth": 34,
    "resnet_shortcut": "A",
    "wide_resnet_k": 2,
    "resnext_cardinality": 32,
    "manual_seed": 1,
    "scales": [
        1,
        0.84089641525,
        0.7071067811803005,
        0.5946035574934808,
        0.4999999999911653
    ],
    "arch": "resnet-34",
    "mean": [
        114.7748,
        107.7354,
        99.475
    ]
}
